src/pages/user/schema.json

```diff
@@ -0,0 +1,70 @@
+{
+	"type": "object",
+	"properties": {
+		"email": {
+			"type": "string",
+			"title": "Correo electrónico",
+			"format": "email",
+		  	"pattern": "^\\S+@\\S+\\.\\S+$",
+			"minLength": 6,
+			"maxLength": 127
+		},
+		"name": {
+			"type": "string",
+			"title": "Nombre",
+			"maxLength": 127
+		},
+		"surname": {
+			"type": "string",
+			"title": "Apellido",
+			"maxLength": 127
+		},
+	  	"birthdate": {
+			"type": "string",
+		  	"title": "Fecha de nacimiento"
+	  	},
```

Comment:
Cuidado que esto no controla que sea una fecha, ver https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times